lib/main.js

```diff
@@ -4,6 +4,8 @@
 import {Implementation} from './Implementation.js';
 import {implementerFiles} from '../implementations/index.js';
 export {Endpoint} from './Endpoint.js';
+// used to ensure that tests are failing from HTTP Errors
+export {HTTPError} from 'ky';
```

Comment:
I would expect this to fail if the `ky` versions don't match up, etc. This is probably a brittle way to do this without having a number of other dependency guarantees in place that might get messy. It would be better to do the assertions based on data if possible.

Reply:
Looking at the source for the Error: https://github.com/sindresorhus/ky/blob/2049affe2ca939f53350fb9b7354c3ff8d2e1c1f/source/errors/HTTPError.ts it seems like we can assert that `response` is there and the `name` is `HTTPError` so yeah can add assertions for it. I was actually putting it there as this lib controls the version of `http-client` we use there for it should be able to replace that error with some other error if the client changes in the future.